WordPress/src/main/java/org/wordpress/android/ui/media/services/MediaUploadService.java

```diff
@@ -39,6 +44,8 @@
     private List<MediaModel> mQueue;
     private List<MediaModel> mCompletedItems;
 
+    private Set<AnalyticsTracker.Stat> availableAnalitycs;
```

Comment:
We could simplify things by using:
```
private final Set<AnalyticsTracker.Stat> mAvailableAnalytics = EnumSet.of(
            AnalyticsTracker.Stat.MEDIA_UPLOAD_STARTED,
            AnalyticsTracker.Stat.MEDIA_UPLOAD_ERROR,
            AnalyticsTracker.Stat.MEDIA_UPLOAD_SUCCESS,
            AnalyticsTracker.Stat.MEDIA_UPLOAD_CANCELED);
```
(Note the typo in the variable name has been corrected, too)